scripts/lint.sh

```diff
@@ -96,6 +97,12 @@ if [[ -s "$TEMP" ]]; then
 fi
 OBJC_STATUS=$?
 
-if [[ $CPP_STATUS != 0 || $OBJC_STATUS != 0 ]]; then
+# Checking forbidden patterns.
```

Comment:
This is better checked in `cpplint.py` itself (where we already have several custom checks). See [`CheckIncludeLine`](https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/cpplint.py#L4533) changes added in #1048.

Reply:
done